ascify art/Ascify-Art.py

```diff
@@ -453,18 +486,14 @@ def change_font(font):
     global selected_font
     selected_font = font
     operation()
-    
+
 def populate(frame):
     # load system fonts  (not all at once because it will lag)
     global loaded_fonts, all_fonts, l
     all_fonts = matplotlib.font_manager.findSystemFonts()
 
-    if len(all_fonts)<100:
-        l = len(all_fonts)
-    else:
-        l = 50
-        
-    for filename in sorted(all_fonts)[:l]: 
+    l = len(all_fonts) if len(all_fonts)<100 else 50

```

Comment:
Function `populate` refactored with the following changes:

- Replace if statement with if expression